activities/RAVLT/items/ravlt_stimulus_1.jsonld

```diff
@@ -0,0 +1,36 @@
+{
+  "@context": [
+    "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/contexts/generic.jsonld"
+  ],
+  "@type": "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/schemas/Field.jsonld",
+  "@id": "ravlt_stimulus_1",
+  "skos:altLabel": "RAVLT Stimulus 1",
+  "skos:prefLabel": "RAVLT Stimulus 1",
+  "schema:description": "Instructions for the Rey Auditory Verbal Learning Test",
+  "schema:schemaVersion": "0.0.1",
+  "schema:version": "0.0.1",
+  "question": {
+    "en": "Press the Play button below to listen to a list of words."
+  },
+  "ui": {
+    "inputType": "audioStimulus"
+  },
+  "inputs": [
```

Comment:
Instead of "inputs" it should be "responseOptions"

Reply:
I've been using `inputs` where the values relate to the configuration of the question, as opposed to the range of acceptable answers. Is this not right? My impression was that `valueConstraints` should relate to constraints on the answer the client can return.

Reply:
maybe we should have `responseOptions` and `inputOptions` ?